pandas/core/reshape/reshape.py

```diff
@@ -50,21 +50,21 @@ class _Unstacker(object):
     ...                                    ('two', 'a'), ('two', 'b')])
     >>> s = pd.Series(np.arange(1.0, 5.0), index=index)
     >>> s
-    one  a   1
-         b   2
-    two  a   3
-         b   4
+    one  a    1.0
+         b    2.0
```

Comment:
another option would be to change the series construction, use `np.arange(1, 5)` instead of `np.arange(1.0, 5.0)`.

I think using integers makes the example slightly simpler